src/main/java/net/lingala/zip4j/model/ExcludeFileHandler.java

```diff
@@ -0,0 +1,9 @@
+package net.lingala.zip4j.model;
+
+import java.io.File;
+
+public interface ExcludeFileHandler {
+
+    boolean isExcluded(File file);
```

Comment:
What about more than one excluded file or directory?

``` 
- root
     -- firstFolder
     -- secondFolder
     -- thirdFolder
     -- firstFile
     -- secondFile
```
For example, I want to exclude firstFolder, secondFolder and firstFile. 
I think I am not able to do it with current solution, right?